com.chartboost.helium.demo/Assets/Samples/Helium SDK/4.0.0/Helium/Editor/HeliumDependencies.xml

```diff
@@ -7,7 +7,7 @@
         </repositories>
 
         <!-- Helium Android SDK -->
-        <androidPackage spec="com.chartboost:helium:3.3.+" />
+        <androidPackage spec="com.chartboost:helium:4.0+" />
```

Comment:
will it make it easier if you went with `4.+`?? 🤔

Reply:
But, if it keeps things clear, it should be fine.

Reply:
Oh, you are correct, was thinking on Canary. 👍🏼

Reply:
4.0+ means any 4.x, right? so 4.1 will work. we probably want 4.0.+
or does that mean 4.09 works but 4.1 does not 🤔